### DISCUSSION TITLE
Rename WLAN name of Pi Pico W

### DISCUSSION THREAD

**Discussion by MarkuBu** [micropython/micropython#9512] [Using MicroPython] [answered]
The Pi Pico W appears as PYBD on my router. How can I give a board a specific name because I will have multiple Pico W in the near future?

**Comment by MarkuBu** [comment]
I found some similar questions and it seams that this is not implemented. That's bad news because then I will have about 10 PYBD listed in my router

**Comment by jimmo** [comment, marked as answer]
@MarkuBu I am working on this right now, hoping to get it in for v1.20.

**Reply by MarkuBu** [reply]
Perfect, thank you
